Initial cut.

timlesallen opened the pull request.

aeh committed:
wip: basic working version

aeh committed:
tests

timlesallen commented on src/get-password-feedback.js:
```diff
@@ -0,0 +1,33 @@
+/**
+ * Use the globally available zxcvbn lib to calculate a passwords strength.
```
```suggestion
 * Use the globally available zxcvbn lib to calculate a password's strength.
```

timlesallen commented on src/load-script-once.js:
```diff
@@ -0,0 +1,23 @@
+  return scriptCache[src];
+}
+
+export default loadScriptOnce;
```
I would have thought someone had already done this exact thing on npm?

timlesallen replied:
In fact, you've got this on package.json? https://www.npmjs.com/package/load-script-once

aeh replied:
@timlesallen I know and I did try using that but it does not have the ability to pass options through... so if I was to use that we could not set the cors anonymous or integrity hash attributes.

timlesallen replied:
@aeh Ah!  Maybe publish yours then? :P

aeh committed:
get code coverage working

aeh committed:
Update src/get-password-feedback.js

Co-authored-by: Tim Allen <tim@noblesamurai.com>

aeh committed:
cleanup a little to make codeclimate happy

aeh committed:
update readme and figure out how to remove .default req

aeh committed:
fix test fixture

aeh committed:
use rollup rather than webpack

aeh committed:
run tests for esm and global function versions

aeh committed:
replace async/await with promises

aeh committed:
remove babel regenerator runtime stuff

aeh committed:
change build targets to "> 0.25%, not dead"

Using targets:
{
  "chrome": "49",
  "edge": "18",
  "firefox": "68",
  "ie": "11",
  "ios": "11.3",
  "opera": "68",
  "safari": "13",
  "samsung": "11.1"
}

aeh committed:
go back to umd (rather than iife)

not much difference in size and we may as well support commonjs require
process for any projects that may want to build things that way rather
than with es modules

aeh committed:
fix titles

aeh committed:
add esm.min version and update readme

aeh committed:
use local version of zxcvbn.js for tests

aeh committed:
add note on how to create the checksum

aeh committed:
add screenshot

aeh committed:
1.0.0-0

timlesallen commented:
@aeh LGTM!
